plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -111,6 +111,9 @@ func init() {
 			Weight: 1,
 		},
 	)
+
+	// Fit is determined by node memory pressure condition.
+	factory.RegisterFitPredicate("NodeMemoryPressure", predicates.NodeMemoryPressurePredicate)
```

Comment:
You should either put this under defaultPredicates() in GeneralPredicates(). Putting it where you did, it is disabled by default.

Put in defaultPredicates() if you only want it to run in the scheduler; put it in GeneralPredicates() if you want it to run in the scheduler and kubelet admission controller.